rules/allied-structures.yaml

```diff
@@ -240,6 +240,8 @@ gaairc:
 		TargetTypes: Ground, C4, SpyInfiltrate, DetonateAttack
 	Power:
 		Amount: -50
+	RepairsUnits:
```

Comment:
Pretty sure RA2 didn't have this. IIRC players could use an Engineer-IFV to repair aircrafts.